lyricsbot/domains/lyricsondemand/config.py

```diff
@@ -0,0 +1,4 @@
+"""
+Format URL link for download.
```

Comment:
Is it file for url download constant or for config? Confused doc-string.

Reply:
Done.